src/context/directory/handlers/emailTemplates.ts

```diff
@@ -22,17 +27,24 @@ function parse(context) {
     if (ext === '.html') sorted[name].html = file;
   });
 
-  const emailTemplates = [];
-  Object.values(sorted).forEach((data) => {
-    if (!data.meta) {
-      log.warn(`Skipping email template file ${data.html} as missing the corresponding '.json' file`);
-    } else if (!data.html) {
-      log.warn(`Skipping email template file ${data.meta} as missing corresponding '.html' file`);
+  const emailTemplates = Object.values(sorted).flatMap(({
```

Comment:
Am assuming `flatMap` is ok because you don't support Node 10

Reply:
We haven't taken a hard stance on Node version support, but yes, based on [it's official support](https://endoflife.date/nodejs), it' safe to say that we're not supporting Node 10.

Although, now that you bring it up, it's certainly an important thing to consider. Everything transpiles into ES2016, but we'll have to double check how many previous versions of Node can adequately run that.